include/zephyr/bluetooth/classic/a2dp.h

```diff
@@ -687,6 +687,7 @@ struct bt_a2dp_stream_ops {
  *
  * Register Audio callbacks for a stream.
  *
+ * @note Registering the same callback struct twice is not supported.
```

Comment:
Same as `bt_aics_client_cb_register`

Reply:
This comment is still unresolved. Could remove the change here as well, as registering the same CB for the same stream is simply a no-op